client/src/pages/reports/Show.js

```diff
@@ -604,40 +603,6 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }) => {
                   />
                 )}
 
-                <Field
-                  name="location"
-                  component={FieldHelper.ReadonlyField}
-                  humanValue={
-                    report.location && (
-                      <LinkTo modelType="Location" model={report.location} />
-                    )
-                  }
-                />
-
-                {report.cancelled && (
-                  <Field
-                    name="cancelledReason"
-                    label="Cancelled Reason"
-                    component={FieldHelper.ReadonlyField}
-                    humanValue={utils.sentenceCase(report.cancelledReason)}
-                  />
-                )}
-
-                {!report.cancelled && (
-                  <Field
-                    name="atmosphere"
-                    label={Settings.fields.report.atmosphere}
-                    component={FieldHelper.ReadonlyField}
-                    humanValue={
-                      <>
-                        {utils.sentenceCase(report.atmosphere)}
-                        {report.atmosphereDetails &&
-                          ` – ${report.atmosphereDetails}`}
-                      </>
-                    }
-                  />
-                )}
-
```

Comment:
Attachments are using the wrong label; should be `Settings.fields.report.attachments.label` instead of `Settings.fields.attachment.shortLabel` (also change this in `reports/Form.js`!).